src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import styles from './TextArea.st.css';
+import {
+  TEXT_AREA_SUCCESS_ICON,
+  TEXT_AREA_DATA_HOOK,
+  TEXT_AREA_ERROR_ICON,
+  TEXT_AREA_ERROR_DESCRIPTION,
+} from './dataHooks';
+// tslint:disable-next-line:no-duplicate-imports
+import { ChangeEvent } from 'react';
```

Comment:
No need to import this.
You can use `React.ChangeEvent`.
That's what we do in other components.